internal/anelace/argparse.go

```diff
@@ -98,7 +98,7 @@ func NewFromArgv(argv []string) (anl *Anelace) {
 		cfg: config{
 			CidMultibase: "base32",
 			HashBits:     256,
-			AsyncHashers: runtime.NumCPU() * 2, // SANCHECK yes, this is high: seems the simd version knows what to do...
+			AsyncHashers: 0, // disabling async hashers for now
```

Comment:
Line 99 - sure don't wanna switch to `base36` ? :) 
( commenting here because UX )

Reply:
done

Reply:
done